go.mod

```diff
@@ -112,6 +112,7 @@ require (
 	github.com/docker/distribution v2.8.3+incompatible // indirect
 	github.com/docker/docker-credential-helpers v0.8.2 // indirect
 	github.com/docker/go-events v0.0.0-20190806004212-e31b211e4f1c // indirect
+	github.com/dustin/go-humanize v1.0.1 // indirect
```

Comment:
I don't think it's an indirect dependency. Please run `make go-mod-tidy` to fix it.